src/Conversion/ONNXToTorch/CMakeLists.txt

```diff
@@ -14,6 +14,9 @@ add_onnx_mlir_library(OMONNXToTorch
   NN/GlobalAveragePool.cpp
   NN/ReduceMean.cpp
   NN/Gemm.cpp
+  NN/SoftmaxOp.cpp
+  NN/ConcatOp.cpp
+  NN/AbsOp.cpp
```

Comment:
@ramakota-helprack, please remove `abs`. I've added it as part of the template PR.

Reply:
Resolved